index.js

```diff
@@ -27,6 +27,10 @@ var GulpJsonServer = function(options){
 			return this.instance;
 		}
 
+		if(this.options.rewriteRules){
```

Comment:
@mijamo I guess this should be after server instance is created in the code below.